client/my-sites/upgrades/domain-search/domain-search.jsx

```diff
@@ -41,10 +41,14 @@ var DomainSearch = React.createClass( {
 	componentDidMount: function() {
 		this.props.sites.on( 'change', this.checkSiteIsUpgradeable );
 		this.props.fetchSitePlans( this.props.sitePlans, this.props.sites.getSelectedSite() );
+
+		this.previousSelectedSite = this.props.sites.getSelectedSite();
```

Comment:
IMO, it would be nicer to move the `sites` observer to the parent component and pass `site` as a prop to this component